v2/spiffebundle/bundle.go

```diff
@@ -0,0 +1,43 @@
+package spiffebundle
+
+import (
+	"crypto/x509"
+	"io"
+
+	"gopkg.in/square/go-jose.v2"
+)
+
+// Bundle is a SPIFFE bundle.
+type Bundle struct {
+	jose.JSONWebKeySet
+
+	Sequence    uint64 `json:"spiffe_sequence,omitempty"`
+	RefreshHint int    `json:"spiffe_refresh_hint,omitempty"`
+}
+
+// Load loads a Bundle from a file on disk.
+func Load(path string) (*Bundle, error) {
```

Comment:
What about Write, Save, or similar?

When I think about loading a bundle from disk, I'm inclined to say that it will be more common to want to load from a set of CA certificates on disk, or a JWKS on disk, than it will be to load an actual SPIFFE bundle from disk. With that in mind, it would probably be useful to have `LoadFromX509Roots` and `LoadFromJWKS` counterparts to go along with this.

Reply:
`LoadFromX509Roots`, etc. sound like a convenient additions.